app/views/surveys/_form.html.haml

```diff
@@ -0,0 +1,11 @@
+= form_for(@survey) do |f|
+  %div.survey-form-title
```

Comment:
`%div.survey-form-title` can be written as `.survey-form-title` since `%div` is implicit